### PR TITLE
bpo-42253: Update xml.dom.minidom.rst

### PR DESCRIPTION
**Pull request by jedie**
The `standalone` arguments was added in Python 3.9. This information is missed in the docu.

https://bugs.python.org/issue42253

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

## Recognized GitHub username

We couldn't find a [bugs.python.org](https://bugs.python.org/) (b.p.o) account corresponding to the following GitHub usernames:

@jedie

This might be simply due to a missing "GitHub Name" entry in one's b.p.o account settings. This is necessary for legal reasons before we can look at this contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.




You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

File: Doc/library/xml.dom.minidom.rst [hunk 1]
```diff
@@ -174,6 +177,9 @@ module documentation.  This section lists the differences between the API and
       The :meth:`toxml` method now preserves the attribute order specified
       by the user.
 
+   .. versionchanged:: 3.9
+      The `standalone` argument was added.
```
Comment (Fidget-Spinner) [inline Doc/library/xml.dom.minidom.rst]:
```suggestion
      The ``standalone`` argument was added.
```

File: Doc/library/xml.dom.minidom.rst [hunk 2]
```diff
@@ -190,6 +196,8 @@ module documentation.  This section lists the differences between the API and
       The :meth:`toprettyxml` method now preserves the attribute order specified
       by the user.
 
+   .. versionchanged:: 3.9
+      The `standalone` argument was added.
```
Comment (Fidget-Spinner) [inline Doc/library/xml.dom.minidom.rst]:
```suggestion
      The ``standalone`` argument was added.
```

File: Doc/library/xml.dom.minidom.rst [hunk 3]
```diff
@@ -156,6 +156,9 @@ module documentation.  This section lists the differences between the API and
       The :meth:`writexml` method now preserves the attribute order specified
       by the user.
 
+   .. versionchanged:: 3.9
+      The `standalone` argument was added.
```
Comment (Fidget-Spinner) [inline Doc/library/xml.dom.minidom.rst]: For reStructuredText in cpython, code styling is usually applied with double backticks.
```suggestion
      The ``standalone`` argument was added.
```

**Review by Fidget-Spinner — COMMENTED** [review]
Hello, thank you for your contribution to CPython!

For other reviewer's reference: the change being documented is found in commit dc04a0571e362cd3de040771d7705cb107ae26fc .

The version it was added to is indeed 3.9. This LGTM, other than the minor backtick nitpicks. The backticks are causing the Travis CI build to fail too.

**Comment by BoboTiG** [comment]
I just stumble upon the lack of documentation (I was thinking the keyword was always available as there was no `Version added/changed:` note :)

@jedie do you mind applying changes and fix conflicts :pray: ?

File: Doc/library/xml.dom.minidom.rst (hunk 3 above)
Comment (serhiy-storchaka) [reply Doc/library/xml.dom.minidom.rst]: Or format it as parameter:
```suggestion
      The *standalone* parameter was added.
```

File: Doc/library/xml.dom.minidom.rst (hunk 1 above)
Comment (serhiy-storchaka) [reply Doc/library/xml.dom.minidom.rst]:
```suggestion
      The *standalone* parameter was added.
```

File: Doc/library/xml.dom.minidom.rst (hunk 2 above)
Comment (serhiy-storchaka) [reply Doc/library/xml.dom.minidom.rst]:
```suggestion
      The *standalone* parameter was added.
```

**Review by serhiy-storchaka — APPROVED** [review]

**Pull request merged**